<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Import Settings</string>
    </property>
    <widget class="QPushButton" name="importSpec_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Spectrum 
File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importBck_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Background 
File</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="subtract_bck_checkBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Subtract Background</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importWLRef_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>White Light
Ref File</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="WLRef_checkBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Correct for White Light</string>
     </property>
    </widget>
   </widget>
   <widget class="PlotWidget" name="plot">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>961</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fitting Settings</string>
    </property>
    <widget class="QComboBox" name="fitFunc_comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="fit_pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_pushButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>620</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Clear Plot</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="norm_checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Normalize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_fig_pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>620</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export Publication
Ready Figure</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
